person.rb

```diff
@@ -1,5 +1,7 @@
+require_relative './decorators/nameable'
 class Person
```

Comment:
- [ ] Person class should inherits from Nameable, please 🙏🏽 use inheritance. We use inheritance as a way to extract common behaviors from classes that share that behavior and move it to a superclass. This lets us keep logic in one place.